py4cast/models/vision/unetrpp.py

```diff
@@ -297,14 +344,16 @@ def __init__(
         self,
         input_size=[32 * 32 * 32, 16 * 16 * 16, 8 * 8 * 8, 4 * 4 * 4],
```

Comment:
This is out of the scope of this PR but could we imagine using powers of 2 to define `input_size` and `dims` ?

Reply:
We could, it is coming straight from the original repo/implementation.